app/main.py

```diff
@@ -1,8 +1,17 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = [Person(person["name"], person["age"]) for person in people]
+    for i in range(len(result)):
+        if "wife" in people[i] and people[i]["wife"] is not None:
+            result[i].wife = Person.people[people[i]["wife"]]
+        if "husband" in people[i] and people[i]["husband"] is not None:
+            result[i].husband = Person.people[people[i]["husband"]]
```

Comment:
Better to iterate through the element, not by their number

Reply:
Changed it, also pls check another task too. https://github.com/mate-academy/py-car-wash-station/pull/1124